app/src/main/java/ca/marshallasch/veil/DataStore.java

```diff
@@ -321,6 +345,10 @@ public void insertSync(@Nullable Sync.SyncMessage syncMessage) {
         hashTableStore.save();
     }
 
+    public void updatePostStatus(int status){
```

Comment:
This function is not used, and its functionality is also covered by `setPostStatus`

Reply:
resovled